src/Middleware/HttpLogging/src/HttpRequestLog.cs

```diff
@@ -4,12 +4,15 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.AspNetCore.HttpLogging
 {
     internal sealed class HttpRequestLog : IReadOnlyList<KeyValuePair<string, string?>>
     {
+        public static ReadOnlySpan<char> RequestSpan => new char[] { 'R', 'e', 'q', 'u', 'e', 's', 't', ':' };
```

Comment:
This allocates every access, the static `ReadOnlySpan` accessor optimization currently only applies to byte[], sbyte[], and bool[].

Reply:
When the ValueStringBuilder isn't used, these should be plain string-constants, then copied via https://github.com/dotnet/runtime/pull/53246 to the destination.